src/popup.svelte

```diff
@@ -0,0 +1,106 @@
+<script lang="ts">
+  import { onMount } from 'svelte';
+
+  import { Constants } from './constants';
+  import type { Config } from './constants';
+  import { PlantUmlEncoder } from './encoder/plantUmlEncoder';
+
+  let config: Config = { ...Constants.defaultConfig };
+  let inputUrl = '';
+  let versionPumlSrc = '';
+  let errorMessage = '';
+  let loading = false;
+
+  onMount(() => {
+    chrome.runtime.sendMessage({ command: Constants.commands.getConfig }, (initialConfig: Config) => {
```

Comment:
The initilization seems run asynchronously. Is it okay to use such an initilization ib onMount?
(e.g. there is a possibility where some variables are still uninitialized after onMount is finished.)

Reply:
hm, it seems okay.
https://svelte.dev/tutorial/onmount